armi/reactor/components/basicShapes.py

```diff
@@ -71,6 +71,9 @@ def __init__(
     def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
         return max(self.getDimension("id", Tc, cold), self.getDimension("od", Tc, cold))
 
+    def getBoundingCircleInnerDiameter(self, Tc=None, cold=False):
+        return min(self.getDimension("id", Tc, cold), self.getDimension("od", Tc, cold))
```

Comment:
Sorry, I don't follow.   Why does this method and the "outer" version use min/max?

Should "id" be min and "od" be max?

Just wondering what I don't understand.

Reply:
This is great question. I figured there was a reason for using `max()` in the "outer" method so I just followed suit for the `min()` case. Happy to change both if there is support for it.

Reply:
hmm...

I'm not thrilled with not know why things are the way they are.  hmmm

Reply:
@ntouran @jakehader @onufer  do you have any idea why `getBoundingCircleOuterDiameter()` uses `max()`?

Reply:
Sometimes when you define components between solid interfaces that have vacuums, they have negative area, and their ID becomes greater than OD because ID is linked to one component and OD to another, and those components overlap.

Reply:
Thanks @onufer !